code/optimize/src/androidTest/java/com/adobe/marketing/mobile/optimize/OptimizeFunctionalTests.java

```diff
@@ -2194,6 +2194,432 @@ public void testPropositionGenerateReferenceXdm() throws IOException {
                 "de03ac85-802a-4331-a905-a57053164d35", decisioning.get("propositionID"));
     }
 
+    // 19
+    @Test
+    public void testGetPropositions_ScopesFromEventIsUpdateInProgress()
```

Comment:
nit: Can we name is to describe what happens in the test? `testGetPropositions_multipleUpdatePropositonsCallsBeforeGetPropositions`

Reply:
Sure, makes more sense 👍